front-end/src/reportWebVitals.js

```diff
@@ -1,13 +0,0 @@
-const reportWebVitals = onPerfEntry => {
```

Comment:
Not sure if we want to delete this

Reply:
ha, you guys prob notice I don't like to keep extra stuff... do you plan on using those web analytics on this project? I think GCP also has a comprehensive analysis tool, prob better than this one. I'll add it back if we're going to use it, but think we should remove if its not being used.